rtf

```diff
@@ -321,6 +327,49 @@ class Platform:
     print("export RUDDER_TOKEN=" + os.environ['RUDDER_TOKEN'])
     print("alias rcli='./rudder-cli --skip-verify --url=" + rudder_url + " --token=" + token + "'")
 
+  def export_test(self, rule_uuid, test_name, scenario=False):
+    """ Export a given rule, for use in a test or a scenario """
+    # Retrieve data
+    (rudder_url, token) = self.get_server_info()
+    endpoint = RudderEndPoint(rudder_url, token, verify=False)
+    rule = endpoint.rule_details(rule_uuid)['rules'][0]
+    directives = []
+    for directive_uuid in rule['directives']:
+      directive = endpoint.directive_details(directive_uuid)['directives'][0]
+      del directive['id']
+      directives.append(directive)
+
+    # create test files
+    rule_file = "spec/tests/"+test_name+"_rule.rb"
+    test_file = "spec/tests/"+test_name+"_test.rb"
+    scenario_file = "scenario/"+test_name+".py"
+
+    make_rule_testfile(rule_file, rule, directives)
+    make_user_testfile(test_file)
+    if scenario:
```

Comment:
Don't you think this should be:

    print """A test file to add the rule via the API has been created in %(rule_file)s
    This file is where you can make change to the generated rule or directive, but it works as is.

    A generic test file to test if the rule has been properly applied has been created in %(test_file)s
    It contains demo code, but since we don't know what the rule does it doesn't contain code.
    -> Pleas edit %(rule_file)s !""" % {variables you need}
    if scenario:
      make_scenario_file(scenario_file, test_name)
      print("Additionnaly, a scenario has been created in " + scenario_file)
    else:
      print("""Add the test to an existing scenario:
      - Add the following line before the call to wait_for_generation on all agents
        run('localhost', '%(test_name)s_rule', Err.BREAK, NAME="Test %(test_name)s", GROUP="special:all")
      - Add the following line after the call to wait_for_generation
        run_on_agents('%(test_name)s_test', Err.CONTINUE)
      - Add the following lines before the removal of the agent nodes
        run('localhost', 'directive_delete', Err.FINALLY, DELETE="%(directive_name)s", GROUP="special:all")
        run('localhost', 'rule_delete', Err.FINALLY, DELETE="%(rule_name)s", GROUP="special:all")
      """ % { 'rule_file': rule_file, 'test_file': test_file, 'test_name': test_name, 
             'directive_name': directive['displayName'], 'rule_name': rule['displayName'] 
            } )